src/main/kotlin/ui/strings/Strings.kt

```diff
@@ -2,124 +2,144 @@ package ui.strings
 
 import io.MusicXml.MUSIC_XML_VERSION
 import ui.strings.Language.English
+import ui.strings.Language.French
 import ui.strings.Language.Japanese
 import ui.strings.Language.Russian
 import ui.strings.Language.SimplifiedChinese
 
-enum class Strings(val en: String, val ja: String, val zhCN: String, val ru: String) {
+enum class Strings(val en: String, val fr: String, val ja: String, val zhCN: String, val ru: String) {
```

Comment:
Could you move `fr` to the last one, so that the order is the same as the following assignments